src/main/java/uk/co/fivium/els/categories/waterheaters/model/ConventionalWaterHeatersForm.java

```diff
@@ -0,0 +1,83 @@
+package uk.co.fivium.els.categories.waterheaters.model;
+
+import uk.co.fivium.els.categories.common.StandardTemplateForm30Char;
+import uk.co.fivium.els.model.meta.FieldPrompt;
+
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+public class ConventionalWaterHeatersForm extends StandardTemplateForm30Char {
+
+  @FieldPrompt("Declared load profile")
+  @NotBlank(message = "Select a declared load profile")
+  private String declaredLoadProfile;
+
+  @FieldPrompt("Water heating energy efficiency class")
+  @NotBlank(message = "Select an energy efficiency indicator")
+  private String efficiencyRating;
+
+  @FieldPrompt("Annual electricity consumption in kWh/annum")
+  @Digits(integer = 4, fraction = 0, message = "Enter the annual electricity consumption, up to 4 digits long")
+  private String kwhAnnum;
+
+  @FieldPrompt("Annual fuel consumption in GJ/annum")
+  @Digits(integer = 2, fraction = 0, message = "Enter the annual fuel consumption, up to 2 digits long")
+  private String gjAnnum;
+
+  @FieldPrompt("Sound power level, indoors dB")
+  @Digits(integer = 2, fraction = 0, message = "Enter the indoors sound power level, up to 2 digits long")
+  private String soundPowerLevelIndoors;
+
+  @FieldPrompt("Is the heater able to only operate during off-peak hours?")
```

Comment:
For heat pump water heaters i changed this to `Can the heat pump be set to work only during off-peak hours?` since I think its a clearer question. Would probably be good to use the same wording here, unless @matteason has any other suggestions?